packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0+4
+
+Adds the `identifier` and `type` fields to the `SKProductDiscountWrapper` to reflect the changes in the [SKProductDiscount](https://developer.apple.com/documentation/storekit/skproductdiscount?language=objc) in iOS 12.2.
```

Comment:
This line needs a bullet (`* `) and please wrap the message at 80 characters:
```suggestion
* Adds the `identifier` and `type` fields to the `SKProductDiscountWrapper` to
reflect the changes in the [SKProductDiscount](https://developer.apple.com/documentation/storekit/skproductdiscount?language=objc) 
in iOS 12.2.
```

Reply:
Done